pixl_dcmd/tests/test_main.py

```diff
@@ -155,3 +156,37 @@ def test_merge_multiple_tag_schemes():
     expected = merge_tag_schemes([base_tag_ops_file])
     result = merge_tag_schemes([base_tag_ops_file, base_tag_ops_file])
     assert result == expected
+
+
+def test_nested_private_tag_deleted(monkeypatch):
+    """
+    GIVEN a dicom image that has a sequence tag (marked keep) with a private tag (marked delete)
```

Comment:
tbf we don't have any sequence tags marked as keep, so could just remove all sequences at this point, and keep this as xfail until we have a user who needs sequences